packages/ui/src/components/molecules/Breadcrumb.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { FC, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+import { DropdownItem, ListItemDropdown } from '..';
+import { DropdownArrow } from '../../assets';
+import { Flex, Typography } from '../atoms';
+
+interface BreadcrumbProps {
+  currentPage: string;
+  breadcrumb?: string;
+  dropdown?: DropdownItem[];
+}
+
+const CustomTypography = styled(Typography)`
+  color: ${({ theme }) => theme.palette.background.breadcrumbSeparator};
+`;
+
+const DropDownWrapper = styled.div`
+  position: relative;
+  cursor: pointer;
+  display: flex;
+  gap: 12px;
+  align-items: center;
+`;
+
+const DropdownContainer = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+`;
+
+const ListItemWrapper = styled.div`
+  display: inline-flex;
+  position: absolute;
+  z-index: 1000;
+  padding: 24px 0px;
+  flex-direction: column;
+  align-items: flex-start;
+  border-radius: 8px;
+  background: ${({ theme }) => theme.palette.background.separatorSecondary};
+  width: 276px;
+  height: 240px;
+  top: calc(100% + 10px);
+  left: 0;
+`;
+
+export const Breadcrumb: FC<BreadcrumbProps> = ({
+  currentPage,
+  breadcrumb,
+  dropdown,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [focusedIndex, setFocusedIndex] = useState(-1);
+  const dropdownState = () => setIsOpen(!isOpen);
+  const [selectedItem, setSelectedItem] = useState('');
```

Comment:
The dropdown state should be in props, just like any other input element

Reply:
done